R/infrastructure.R

```diff
@@ -108,7 +109,14 @@ renv_infrastructure_write_activate <- function(project = NULL,
     return(FALSE)
 
   template <- renv_file_read(source)
-  new <- renv_template_replace(template, list(VERSION = version))
+  new <- renv_template_replace(
+    text = template,
+    replacements = list(
+      version = stringify(c(version)),
+      sha = stringify(c(sha))
```

Comment:
Why `c()`?

Reply:
Maybe `as.character()` would be clearer?

Reply:
Good idea. Even the documentation agrees with you. `?c` has:

>      'c' is sometimes used for its side effect of removing attributes
>      except names, for example to turn an 'array' into a vector.
>      'as.vector' is a more intuitive way to do this, but also drops
>      names.